packages/ethers/src/client.ts

```diff
@@ -244,12 +232,6 @@ export class Web3Modal extends Web3ModalScaffold {
           return Boolean(window.ethereum)
         }
 
-        if (ethersConfig.injected) {
-          if (!window?.ethereum) {
-            return false
-          }
-        }
-
```

Comment:
do we still need this? asking cause this comes from `ethersConfig`

Reply:
We don't need this anymore